deepmd/loss/tensor.py

```diff
@@ -12,24 +12,60 @@ class TensorLoss () :
     def __init__ (self, jdata, **kwarg) :
         try:
             model = kwarg['model']
-            type_sel = model.get_sel_type()
+            self.type_sel = model.get_sel_type()
         except :
-            type_sel = None
+            self.type_sel = None
         self.tensor_name = kwarg['tensor_name']
         self.tensor_size = kwarg['tensor_size']
         self.label_name = kwarg['label_name']
-        self.atomic = kwarg.get('atomic', True)
+        self.atomic = kwarg.get('atomic', None)
         if jdata is not None:
             self.scale = jdata.get('scale', 1.0)
         else:
             self.scale = 1.0
+
+        # YHT: added for global / local dipole combination
+        if self.atomic is True: # upper regulation, will control the lower behavior
+            self.local_weight,self.global_weight = 1.0,0.0
+        elif self.atomic is False: # upper regulation, will control the lower behavior
+            self.local_weight,self.global_weight = 0.0,1.0
+        else: # self.atomic is None, let the loss parameter decide which mode to use
+            if jdata is not None:
+                self.local_weight = jdata.get('atomic_' + self.tensor_name,None)
```

Comment:
It seems that the name of the prefactors are `atomic_dipole` and `dipole`.   

I would suggest to change them to `pref_atomic_dipole` and `pref_dipole`.

Reply:
I will fix this